src/cli/task.rs

```diff
@@ -309,20 +317,38 @@ pub fn execute(args: Args) -> miette::Result<()> {
 
             if available_tasks.is_empty() {
                 eprintln!("No tasks found",);
+            } else if args.summary {
```

Comment:
This will break the `pixi run <TAB TAB>` autocompletion on `bash` and `zsh`. As it expects the output of `pixi task list --summary` to be in a certain format. 

We could possibly move the under a different CLI command as it is not really used by the user, so maybe even hidden.

Reply:
how about `pixi task list --machine` and also hide this command from the help, or do you have another suggestion?

Reply:
have added a hidden argument and have updated the other locations where it's used accordingly.